README.md

```diff
@@ -79,32 +94,136 @@ Support Matrix for Import and Export Operations:
+  --export-pipeline     Execute all export tasks.
+  --import-pipeline     Execute all import tasks.
+  --validate-pipeline   Validate exported data between source and destination.
+  --validate-source-session VALIDATE_SOURCE_SESSION
+                        Session used by exporting source workspace. Only used for --validate-pipeline.
+  --validate-destination-session VALIDATE_DESTINATION_SESSION
+                        Session used by exporting destination workspace. Only used for --validate-pipeline.
+  --use-checkpoint      use checkpointing to restart from previous state
+  --skip-tasks SKIP_TASKS [SKIP_TASKS ...]
+                        List of tasks to skip from the pipeline.
+  --num-parallel NUM_PARALLEL
+                        Number of parallel threads to use to export/import
+  --retry-total RETRY_TOTAL
+                        Total number or retries when making calls to Databricks API
+  --retry-backoff RETRY_BACKOFF
+                        Backoff factor to apply between retry attempts when making calls to Databricks API
+  --start-date START_DATE
+                        start-date format: YYYY-MM-DD. If not provided, defaults to past 30 days. Currently, only used for exporting ML runs objects.
+  --exclude-work-item-prefixes EXCLUDE_WORK_ITEM_PREFIXES [EXCLUDE_WORK_ITEM_PREFIXES ...]
+                        List of prefixes to skip export for log_all_workspace_items
+```
+
+### Exporting the Workspace
+
+To export a workspace, run:
+
+`python3 migration_pipeline.py --profile $SRC_PROFILE --export-pipeline --use-checkpoint [--session $SESSION_ID]`
+
+Where `$SRC_PROFILE` is the Databricks profile for the source workspace, as configured during Setup, and `$SESSION_ID` is an optional session identifier used for subsequent checkpoint runs. All data is exported to a folder named according to the `$SESSION_ID` value under the logs folder - “logs/`$SESSION_ID`”. If `$SESSION_ID` is not specified, a random value will be generated.
+
+#### Recommended parameters and checkpointing
+
+As a starting point, we recommend using the following parameter values:
+* retry-total=30
+* num-parallel=8
+* retry-backoff=1.0
+
+These can be adjusted per your scenario if needed; in general, if API limits are being hit, you can increase `retry-backoff`, decrease `num-parallel`, or both. 
+
+If script failure occurs, you can safely rerun the same command with --use-checkpoint and --session $SESSION_ID to let the migration pick up from the previous checkpoint and rerun.
+
+#### Updating the AWS Account ID
+If your source and destination workspaces are in different accounts, you will need to update the Instance Profile ARN accordingly during the migration. To do this, run the following command after exporting the workspace assets:
+
+```python3 export_db.py --profile $SRC_PROFILE --update-account-id --use-checkpoint --old-account-id $OLD_AWS_ACCT_ID --new-account-id $NEW_AWS_ACCT_ID --session $SESSION_ID
+```
```

Comment:
@gregwood-db @kevinkim9264 I don't think this is going to work. 
* `--new-account-id` is not a parameter, it should be `--update-account-id`
* AFAIK `export_db.py` doesn't support sessions. To work around that you need to set something like `--set-export-dir $EXPORT_PATH/$SESSION_ID`. Or is there any other way?
Check out https://github.com/databrickslabs/migrate/issues/162#issuecomment-1102639987
Other than that thanks for the README update!

Reply:
Thanks @petrcv-db - updated to reflect this.